lale/lib/lale/grid_search_cv.py

```diff
@@ -142,25 +144,15 @@ def predict(self, X):
 _input_fit_schema = {
     'type': 'object',
     'properties': {
-        'X': {
-            'type': 'array',
-            'items': {
-                'type': 'array',
-                'items': {'type': ['number', 'string']}}},
-        'y': {
-            'type': 'array', 'items': {'type': 'number'}}}}
+        'X': {},
+        'y': {}}}
 
 _input_predict_schema = {
     'type': 'object',
     'properties': {
-        'X': {
-            'type': 'array',
-            'items': {
-                'type': 'array',
-                'items': {'type': ['number', 'string']}}}}}
+        'X': {}}}
 
-_output_predict_schema = {
-    'type': 'array', 'items': {'type': 'number'}}
+_output_predict_schema:Dict[str,Any] = {}
```

Comment:
That makes sense, since we don't know a-priori what the output schema will be. In the future, we may need to add a `transform_schemas` method so that the output schema of the grid search is the same as the output schema of the pipeline that it wraps.